index.js

```diff
@@ -7,12 +7,105 @@ function ask(questionText) {
   });
 }
 
+
 start();
 
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
+  console.log("\nLet's play a game where you make up a number and I try to guess it.")
+  let secretNumber = await ask("\nWhat is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
-  process.exit();
+  //secretNumber = the number user inputs for computer to guess.
+  function compGuess(max, min) {
+    //compGuess = Function for Computer generated guess
+    return (min) + Math.floor(Math.random() * (max - min))
+
+  }
+  game()
+
+  async function game() {
+    let max = Number(process.argv[2]) || 100
+    //allows user to input a mx number for the guessing range. If no number inputed will revert to 100
```

Comment:
Your comments throughout your code are good. Comments are generally written on the line above the code it is meant to explain, but other than that, good.